src/main/java/models/Player.java

```diff
@@ -31,4 +57,26 @@ public boolean doubleDown(ArrayList<Card> deck){
             return true;
         }
     }
-}
+
+    public void split(){
+        playerBet = playerBet* 2;
+
+
+        split_hand.add(hand.get(hand.size()-1));
+        hand.remove(hand.get(hand.size()-1));
+        is_split = true;
+    }
+    public void split_hit(ArrayList<Card> deck){
+
```

Comment:
Can I ask how does split_hit() work?
It seems hit() for split_hand, but I don't understand what 

if(is_split == true && getTotal() > 21) {...} else {...}

is doing.  ("getTotal() > 21", "turnEnd = false", and "else hit(deck)")

and also, since stay() is for the original hand, I think you may need a new variable like "split_turnEnd" and a method for that like "split_stay()."